### ISSUE TITLE
Unexpected error message soon after starteing beta3

### ISSUE THREAD

**Issue by vi** [Tribler/tribler#4522]
##### Tribler version/branch+revision: #####

38cb26b662c37b200a8c1d37c947caf6c4a1f485 v7.3.0-beta3

##### Operating system and version: #####

Linux

##### Steps to reproduce the behavior: #####

1. Have a profile from v7.3.0-beta2
2. Upgade git worktree to v7.3.0-beta3
3. `python2 -m pip install aead` to fix obvious error
4. Start the GUI
5. Wait a bit

##### Expected behavior: #####

Works

##### Actual behavior: #####

Crashes

##### Relevant log file output: #####

```
ERROR   1558999426.92                    tunnel:149  (TunnelExitSocket)  Failed to write data to transport: [Errno 22] Invalid argument. Destination: ('xxx.xxx.xxx.xxx', 0) error was: error(22, 'Invalid argument')
INFO    1558999427.45                  provider:43   (DHTCommunityProvider)  Looked up 25440a0c1516370ae0da1914ee6f1e422975736f in the DHTCommunity, got 0 results
Unhandled error in Deferred:


ERROR   1558999427.46            tribler_window:74   (root)  Traceback (most recent call last):
  File "/home/tribler/tribler/TriblerGUI/event_request_manager.py", line 114, in on_read_data
    raise RuntimeError(json_dict["event"]["text"])
RuntimeError: [Failure instance: Traceback: <class 'Tribler.pyipv8.ipv8.messaging.anonymization.tunnelcrypto.CryptoException'>: Error encrypting message for unknown circuit 86898591
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:1355:gotResult
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:1342:_inlineCallbacks
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:457:callback
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:565:_startRunCallbacks
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:651:_runCallbacks
/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/hidden_services.py:305:dht_callback
/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/hidden_services.py:319:send_peers_response
/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py:492:send_cell
/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/payload.py:110:encrypt
]
```

Happend twice now with the same backtrace.

**Comment by egbertbouman** [comment]
Already fixed in latest IPv8, we just need to update the pointer.

**Comment by vi** [comment]
Updated `Triber/pyipv8` to master, now the error is not happening.

But Token Balance stays `-`. If I click on it, I receive `An error occurred during the request: the user with the provided public key could not be found`.

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/twisted/web/server.py", line 190, in process
    self.render(resrc)
  File "/usr/lib/python2.7/dist-packages/twisted/web/server.py", line 241, in render
    body = resrc.render(self)
  File "/usr/lib/python2.7/dist-packages/twisted/web/resource.py", line 250, in render
    return m(request)
  File "/home/tribler/tribler/Tribler/Core/Modules/restapi/trustchain_endpoint.py", line 87, in render_GET
    return json.twisted_dumps({'statistics': self.session.lm.wallets['MB'].get_statistics()})
  File "/home/tribler/tribler/Tribler/Core/Modules/wallet/tc_wallet.py", line 178, in get_statistics
    interacts = self.get_num_unique_interactors(public_key)
  File "/home/tribler/tribler/Tribler/Core/Modules/wallet/tc_wallet.py", line 161, in get_num_unique_interactors
    block_type=b'tribler_bandwidth'):
TypeError: get_latest_blocks() got an unexpected keyword argument 'block_type'
```

**Comment by qstokkink** [comment]
Tribler is not compatible with the latest IPv8 pointer. The `get_latest_blocks()` changed its function signature and #4503 will have to be merged (or the market removed).

**Comment by qstokkink** [comment]
Resolved by #4547
